charts/ocis/templates/appprovider/deployment.yaml

```diff
@@ -63,6 +63,8 @@ spec:
               value: {{ $officeSuite.uri | quote }}
             - name: APP_PROVIDER_WOPI_INSECURE
               value: {{ $officeSuite.insecure | quote }}
+            - name: APP_PROVIDER_WOPI_DISABLE_CHAT
+              value: {{ $officeSuite.disableChat | quote }}
```

Comment:
@wkloucek sorry for stupid question: Will I have to add the `disableChat` flag to collabora also? Or will it assume empty string if it can't find the flag?